src/Main.java

```diff
@@ -16,5 +16,49 @@ public static void main(String[] args){
         System.out.println(cat);
         paper = paper + 4;
         System.out.println(paper);
+
+        dog = dog - 3.5;
+        cat = cat - 1.6;
+        paper = paper - 7639;
+        System.out.println(dog);
+        System.out.println(cat);
+        System.out.println(paper);
+
+        var friend = 19;
+        System.out.println(friend);
+        friend = friend - 2;
+        System.out.println(friend);
+        friend = friend / 7;
+        System.out.println(friend);
+
+        var frog = 3.5;
+        System.out.println(frog);
+        frog = frog * 10;
+        System.out.println(frog);
+        frog = frog / 3.5;
+        System.out.println(frog);
+        frog =  frog + 4;
+        System.out.println(frog);
+
+        var boxer1 = 78.2;
+        var boxer2 = 82.7;
+        var  totalWeight= boxer1 + boxer2;
+        System.out.println(totalWeight);
+        var ves = boxer2 - boxer1;
+        System.out.println(ves);
+
+        var difference = boxer2 % boxer1;
+        System.out.println(difference);
+
+        var totalHours = 640;
+        var time = 8;
+        var totalWrkers = totalHours /time;
+        System.out.println("Всего работников в компании — " + totalWrkers + " человек");
+
+        var bigCompany = totalWrkers + 94;
```

Comment:
```suggestion
        totalWrkers = totalWrkers + 94;
```
можно было не создавать новую переменную, а просто обновить имеющуюся